app/static/src/js/tools.js

```diff
@@ -81,6 +82,10 @@ const roll = (sides) => {
   return Math.floor(Math.random() * sides);
 };
 
+const roll2d6 = () => {
```

Comment:
this could probably benefit from having more generic method, which can parse notation like `2d6`, but for the sake of this simple issue I went with more direct approach.

Reply:
fyi, there is a roll function in utils.js you can import
utils.roll(2, 6)

Reply:
good catch, updated